clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/Role.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.model;
+
+import java.util.Collection;
+
+public interface Role {
```

Comment:
Seems to me like IAM models belong in `clouddriver-aws`.

Reply:
Done.

Moved the associated `RoleProvider` and `TrustRelationship` as well.